tests/test_behav.py

```diff
@@ -641,3 +645,75 @@ def test_action_value_with_chance_player_action(game: gbt.Game, rational_flag: b
     chance_action = game.players.chance.infosets[0].actions[0]
     with pytest.raises(ValueError):
         game.mixed_behavior_profile(rational=rational_flag).action_value(chance_action)
+
+
+@pytest.mark.parametrize(
+    "game,rational_flag,data",
+    [(games.create_mixed_behav_game(), True, [[[0, 1]], [[1, 0]], [[1, 0]]]),
+     (games.create_mixed_behav_game(), True, [[[1/5, 4/5]], [[1/4, 3/4]], [[1, 0]]]),
+     (games.create_complicated_extensive_game(), True, [[[1/5, 4/5], [3/5, 2/5]], [[1/4, 3/4]]]),
+     (games.create_mixed_behav_game(), False, [[[0, 1]], [[1, 0]], [[1, 0]]]),
+     (games.create_mixed_behav_game(), False, [[[1/5, 4/5]], [[1/4, 3/4]], [[1, 0]]]),
+     (games.create_complicated_extensive_game(), False, [[[1/5, 4/5], [3/5, 2/5]], [[1/4, 3/4]]])
+     ]
+)
+def test_specific_profile(game: gbt.Game, rational_flag: bool, data: float):
+    """Test that the mixed behavior profile is initialized from a specific distribution
+    for each player over his actions.
+    """
+    profile = game.mixed_behavior_profile(rational=rational_flag, data=data)
+    for (player, behavior) in zip(game.players, data):
+        assert profile[player] == behavior
+
+
+@pytest.mark.parametrize(
+    "game,rational_flag,data",
+    [(games.create_mixed_behav_game(), True,
+      [[[0, 1, 0]], [[1, 0]], [[1, 0]]]),
+     (games.create_mixed_behav_game(), True,
+      [[[0, 1]], [[1, 0]], [[1, 0]], [[0, 1]]]),
+     (games.create_complicated_extensive_game(), True,
+      [[[1/5, 4/5], [3/5, 2/5]], [[1/4, 3/4], [1/4, 3/4]]]),
+     (games.create_el_farol_bar_game(), True,
+      [[4/9, 5/9], [0], [1/2, 1/2], [11/12, 1/12], [1/2, 1/2]]),
+     (games.create_el_farol_bar_game(), True,
+      [[1/2, 1/2]]),
+     (games.create_mixed_behav_game(), False,
+      [[[0, 1, 0]], [[1, 0]], [[1, 0]]]),
+     (games.create_mixed_behav_game(), False,
+      [[[0, 1]], [[1, 0]], [[1, 0]], [[0, 1]]]),
+     (games.create_complicated_extensive_game(), False,
+      [[[1/5, 4/5], [3/5, 2/5]], [[1/4, 3/4], [1/4, 3/4]]]),
+     (games.create_el_farol_bar_game(), False,
+      [[4/9, 5/9], [0], [1/2, 1/2], [11/12, 1/12], [1/2, 1/2]]),
+     (games.create_el_farol_bar_game(), False,
+      [[1/2, 1/2]])
+     ]
+)
+def test_profile_data_error(game: gbt.Game, rational_flag: bool, data: float):
+    """Test to ensure a pygambit.ValueError is raised when the data do not
+    match with the number of players, the number of the infosets, and the
+    number of actions per infoset.
+    """
+    with pytest.raises(ValueError):
+        game.mixed_behavior_profile(rational=rational_flag,data=data)
```

Comment:
We prefer a space between function arguments (this is done consistently everywhere else)